src/main/java/step3/Controller.java

```diff
@@ -0,0 +1,23 @@
+package step3;
+
+import step3.numbergenerator.NumberGenerator;
+import step3.numbergenerator.RandomNumberGenerator;
+import step3.printer.Printer;
+import step3.printer.PrinterImpl;
+
+public class Controller {
+
+
+    public void main(){
```

Comment:
```suggestion
public class Controller {

    public void main(){
```

개행이 별거 아닌거 같아도, 클린코드의 기본이 되는거 같습니다.